src/Courses/CourseList.tsx

```diff
@@ -13,11 +13,16 @@ export const CourseList: React.FC<CourseListProps> = ({courses}) => {
 
     return (
         <ListGroup>
-            {courses.map(obj => <ListGroupItem action href={`${url}/${obj.id}`} key={`course${obj.id}`}>
-                <CourseRowComponent 
-                    {...obj}
-                />
-            </ListGroupItem>)}
+            {/* TODO: Make a link tag */}
```

Comment:
It is a link?

Reply:
Yeah, this was my way of remembering to do it.